tests/decorators/test_uri_parsing.py

```diff
@@ -0,0 +1,65 @@
+import pytest
+from connexion.decorators.uri_parsing import (AlwaysMultiURIParser,
+                                              FirstValueURIParser,
+                                              Swagger2URIParser)
+
+QUERY1 = ["a", "b,c", "d,e,f"]
+QUERY2 = ["a", "b|c", "d|e|f"]
+PATH1 = "d,e,f"
+PATH2 = "d|e|f"
+CSV = "csv"
+PIPES = "pipes"
+MULTI = "multi"
+
+
+@pytest.mark.parametrize("cls, expected, query_in, collection_format", [
+        (Swagger2URIParser, ['d', 'e', 'f'], QUERY1, CSV),
+        (FirstValueURIParser, ['a'], QUERY1, CSV),
+        (AlwaysMultiURIParser, ['a', 'b', 'c', 'd', 'e', 'f'], QUERY1, CSV),
+        (Swagger2URIParser, ['a', 'b', 'c', 'd', 'e', 'f'], QUERY1, MULTI),
+        (FirstValueURIParser, ['a', 'b', 'c', 'd', 'e', 'f'], QUERY1, MULTI),
+        (AlwaysMultiURIParser, ['a', 'b', 'c', 'd', 'e', 'f'], QUERY1, MULTI),
+        (Swagger2URIParser, ['d', 'e', 'f'], QUERY2, PIPES),
+        (FirstValueURIParser, ['a'], QUERY2, PIPES),
+        (AlwaysMultiURIParser, ['a', 'b', 'c', 'd', 'e', 'f'], QUERY2, PIPES)])
+def test_uri_parser_query_params(cls, expected, query_in, collection_format):
```

Comment:
It's a simple thing, but I would suggest to change the name of the first argument from "cls" to "parser_class", I was very confused seen this code at first due to https://www.python.org/dev/peps/pep-0008/#function-and-method-arguments

Reply:
sure thing - thanks for the review!